databuilder/databuilder/models/table_stats.py

```diff
@@ -10,20 +10,128 @@
 from databuilder.models.graph_node import GraphNode
 from databuilder.models.graph_relationship import GraphRelationship
 from databuilder.models.graph_serializable import GraphSerializable
-from databuilder.models.table_metadata import ColumnMetadata
+from databuilder.models.table_metadata import ColumnMetadata, TableMetadata
 from databuilder.models.table_serializable import TableSerializable
 
+LABEL = 'Stat'
+STAT_RESOURCE_RELATION_TYPE = 'STAT_OF'
+RESOURCE_STAT_RELATION_TYPE = 'STAT'
+
+
+class TableStats(GraphSerializable, TableSerializable):
```

Comment:
What are the stats that we want to surface? There are some metadata on the left side panel in table details page, for example, total number of rows, etc. Will some of the metadata be moved to this stats model?

Reply:
As of right now we are indexing usage stats like dau, query count, and service specific usage like for airflow, flyte, etc. In the future however we could use this to index stats about the data in the table as well, similar to how we already do for columns.